Source/UnitTests/GTMSessionFetcherFetchingTest.m

```diff
@@ -1667,6 +1715,13 @@ - (void)testQuickBeginStopFetching {
 
   // This test exercises the workaround for Radar 18471901. See comments in GTMSessionFetcher.m
   int const kFetcherCreationCount = 1000;
+  XCTestExpectation *startExpectation =
```

Comment:
who fulfulls these?

Reply:
Oh, misread and didn't see the `expectationForNotification:`, was thinking they were regular expectations.